firestore/FirestoreSwiftUIExample/ViewModels/RestaurantViewModel.swift

```diff
@@ -60,9 +103,17 @@ class RestaurantViewModel: ObservableObject {
             return nil
           }
         }
-
-        print(self.reviews)
       }
     }
   }
+
+  func getNSError(document: DocumentSnapshot) -> NSError {
+    return NSError(
+      domain: "AppErrorDomain",
+      code: -1,
+      userInfo: [
+        NSLocalizedDescriptionKey: "Unable to retrieve population from snapshot \(document)"
```

Comment:
`population` seems like the wrong word to use here.

Reply:
Thanks! copy and paste error